src/util.js

```diff
@@ -12,5 +12,15 @@ module.exports = {
     return _.every(collection, function (item) {
       return _.isString(item);
     });
+  },
+
+  isArrayOfArrays: function (collection) {
+    return _.isArray(collection) && _.every(collection, function (item) {
+      return _.isArray(item);
+    });
+  },
```

Comment:
Oh for sure I get that part – I mean instead of the contents of this function being this lodash `isArray` and `every` methods, maybe React has a function we could use so it would look more like:

```
React.doesValidate(collection, React.PropTypes.arrayOf(React.PropTypes.array));
```

...which just seems cleaner and more declarative to me.